web-frontend/src/main.tsx

```diff
@@ -37,6 +38,7 @@ const router = createBrowserRouter(
       <Route path="account-create-step-one" element={<StepOne />} />
       <Route path="account-create-step-two" element={<StepTwo />} />
       <Route path="account-create-step-three" element={<StepThree />} />
+      <Route path="homepage-transfer-send" element={<SendCoins />} />
```

Comment:
The url is: homepage-transfer-send and we are rendering `<SendCoins>`. Please, let's keep a standard for that as also mentioned here: https://discord.com/channels/@me/1100786679486369844/1113023630364901406

Reply:
Fist, why homepage!? Then, if you want to keep your standard it should be `send-coins`. Finally, it's a huge name that does not match with the component name you are rendering.

Change that for: /send-coins and render the `<SendCoins>` component. So, your URL will match with the component you are rendering.

/apple -> </Apple>
/number -> </Number>
/something -> </Something>

and not 

/apple -> </Tomato>

Reply:
understood ! thank you for the clarifications

Reply:
changed !